### ISSUE TITLE
Weight setting

### ISSUE THREAD

**Issue by weiling666** [passalis/probabilistic_kt#3]
loss = cosine_similarity_loss(outputs_net, output_target) + \
                       supervised_weight * supervised_loss(outputs_net, targets)
Could you share this supervised weight setting?THANKS
